awx/api/views/__init__.py

```diff
@@ -752,6 +753,14 @@ class ProjectNotificationTemplatesAnyList(SubListCreateAttachDetachAPIView):
     relationship = 'notification_templates_any'
 
 
+class ProjectNotificationTemplatesStartedList(SubListCreateAttachDetachAPIView):
+
+    model = models.NotificationTemplate
```

Comment:
(see comment below) ...this one doesn't.  Maybe we should update *all* of these views to follow the pattern where they inherit from the `All` view?  Seems cleaner/less code to me.

Reply:
I went ahead and updated this file with that pattern!